src/jit/regalloc.cpp

```diff
@@ -1930,49 +1930,49 @@ regMaskTP           Compiler::rpPredictTreeRegUse(GenTreePtr    tree,
             else if (rpHasVarIndexForPredict(predictReg))
             {
                 /* Get the tracked local variable that has an lvVarIndex of tgtIndex1 */
-
-                unsigned tgtIndex1 = rpGetVarIndexForPredict(predictReg);
-                LclVarDsc * tgtVar   = lvaTable + lvaTrackedToVarNum[tgtIndex1];
-                VarSetOps::MakeSingleton(this, tgtIndex1);
-
-                noway_assert(tgtVar->lvVarIndex == tgtIndex1);
-                noway_assert(tgtVar->lvRegNum   != REG_STK);  /* Must have been enregistered */
+                {
+                    unsigned tgtIndex1 = rpGetVarIndexForPredict(predictReg);
+                    LclVarDsc * tgtVar   = lvaTable + lvaTrackedToVarNum[tgtIndex1];
+                    VarSetOps::MakeSingleton(this, tgtIndex1);
+    
+                    noway_assert(tgtVar->lvVarIndex == tgtIndex1);
+                    noway_assert(tgtVar->lvRegNum   != REG_STK);  /* Must have been enregistered */
```

Comment:
This is not just whitespace, I added a scope (see the additional {}), so that the declaration isn't in the same scope as the label for the goto, can hoist the definitions above the goto if that's preferable.